app/src/androidTest/java/com/github/swent/echo/compose/map/LocationSelectorTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.github.swent.echo.compose.map
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class LocationSelectorTest {
+
+    @get:Rule val composeTestRule = createComposeRule()
+
+    @Test
+    fun shouldDisplayLocationSelector() {
+        composeTestRule.setContent { LocationSelector {} }
```

Comment:
Nitpicking: this could be done in setUp function annotated with @Before

Reply:
Since the content for both tests is different, I would think it's cleaner to leave it in the test itself. Otherwise I'm not sure how to set both contents simultaneously.

Reply:
Yeah, you're right, I didn't read it carefully enough. In this case, it makes no sense to add a setUp function!